deploy/template.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: aws-account-shredder-template
+
+parameters:
+  - name: IMAGE
+    required: true
+    value: quay.io/agautam/aws-account-shredder
+  - name: IMAGE_TAG
+    required: true
+    value: latest
+  - name : REPLICAS
+    required: true
+    value : "1"
+
+objects:
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: aws-account-shredder
+      labels:
+        name: aws-account-shredder
+    spec:
+      replicas: ${REPLICAS}
+      strategy:
+        type: RollingUpdate
+        rollingUpdate:
+          maxUnavailable: 0
+          maxSurge: 1
+      selector:
+        matchLabels:
+          name: aws-account-shredder
+      template:
+        metadata:
+          labels:
+            name: aws-account-shredder
+        spec:
+          serviceAccountName: aws-account-shredder
+          containers:
+            - name: aws-account-shredder
+              image: ${IMAGE}:${IMAGE_TAG}
+              imagepullpolicy: always
```

Comment:
Could you change to IfNotPresent？

Reply:
Can I ask why `IfNotPresent`? Our operators all use `always`

Reply:
It is not strict. IfNotPresent will help to save some time for restarting pod. 
I am ok with both

Reply:
:+1: